modules/ROOT/pages/instanton-limitations.adoc

```diff
@@ -161,6 +162,72 @@ This configuration uses placeholder values for things like the database name, ho
 
 If an application is injected with a `DataSource` before the checkpoint and the configuration of the `DataSource` changes, the application is restarted when the InstantOn application container image is run with the updated configuration. You can avoid this scenario by using the `beforeAppStart` option or by modifying the component not to be early startup code. In this example, that modification is to remove the `loadOnStartup = 1` attribute.
 
+[#mp-config-no-default]
+== Accessing microprofile config with no default value at Checkpoint
+The configuration property can be introduced inside the application either statically through `<T>` or `Optional<T>`, or dynamically using a `Provider<T>`. The following example shows ways to inject static, static-optional, dynamic and dynamic-optional config property. 
```

Comment:
I think we need a brief problem summary to introduce this section- generally, what is the problem or error user might encounter that they will find remedy for in this section?